src/utilities/include/par.h

```diff
@@ -134,9 +134,9 @@ inline ExecutionPolicy autoPolicy(int size) {
         thrust::NAME(thrust::MANIFOLD_PAR_NS::par, args...); \
         break;                                               \
       case ExecutionPolicy::Seq:                             \
+        thrust::NAME(thrust::cpp::par, args...);             \
```

Comment:
Is the same error in the function above? Or is there some reason it works differently there?

Reply:
the function above will return the result, so it cna only fall through for `ExecutionPolicy::Seq`. Also, the reason I need to write it in this weird form instead of putting the last return in the switch case is due to compiler warning, the compiler somehow thinks that policy can fall through and never return anything.